### ISSUE TITLE
Mitsue-Links Accessibility Dept. Comment on SC 3.2.7 Change of Content: Avoid using the term "control"

### ISSUE THREAD

**Issue by kurosawa-takeshi** [w3c/wcag21#506]
The second exception of [SC 3.2.7 Change of Content](https://www.w3.org/TR/2017/WD-WCAG21-20170912/#change-of-content) should avoid using the term "control," because a "change of content" could be triggered by user action without using any controls of a page (e.g. tilting the device).

Currently the second exception is written as below,

> The user has been advised of the change of content before, or as a result of using the control;

Though [SC 2.6.1 Device Sensors](https://www.w3.org/TR/2017/WD-WCAG21-20170912/#device-sensors) requires alternative methods for functionalities which require specific device sensors, users who intentionally use specific device sensors should be advised.

**Comment by DavidMacDonald** [comment]
WCAG 2 uses control  https://www.w3.org/TR/WCAG20/#user-interface-componentdef
Perhaps "triggering mechanism"?

>  The user has been advised of the change of content before, or as a result of using the triggering mechanism;

**Comment by kurosawa-takeshi** [comment]
It is fine for us to use “triggering mechanism”.

**Comment by DavidMacDonald** [comment]
Proposed response: 

We have created a pull request to replace "control that triggers it" with "triggering mechanism"

https://github.com/w3c/wcag21/pull/521

**Comment by awkawk** [comment]
Pull request needs to only address this change, currently it makes a bunch of other changes.

**Comment by DavidMacDonald** [comment]
@awkawk Actually the edit is exactly what it says it is
https://github.com/w3c/wcag21/pull/521/commits/513c371cbcc803e31daa52ca73a428bc9de5cc91
Now other requests are part of that branch. Not sure what to do about that.

**Comment by alastc** [comment]
You're both right: if you make a pull request and then add changes to that branch, it adds them to the change request. 
I would either revert the branch back to the point with just those changes, or do that locally and create a separate branch to apply the edits.

**Comment by joshueoconnor** [comment]
Survey https://www.w3.org/2002/09/wbs/35422/Oct_17th_agwg/results

**Comment by DavidMacDonald** [comment]
Amended response as per meeting oct 17.
First bullet would become 
* There is a programmatically determined relationship between the new content and triggering mechanism
The second would become
* The user has been advised of the change of content before, or as a result of taking the action

**Comment by joshueoconnor** [comment]
RESOLUTION: Accept David’s 2 edits. https://www.w3.org/2017/10/17-ag-minutes.html
